### PR TITLE
imhex: new, 1.35.4

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- imhex: bump REL for topic Revision Marking Guidelines
- imhex: new, 1.35.4

Package(s) Affected
-------------------

- imhex: 1.35.4-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit imhex
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-editors/imhex/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        -DUSE_SYSTEM_BOOST=ON \
+        -DIMHEX_ENABLE_LTO=ON \
+        -DIMHEX_STRIP_RELEASE=OFF \
+        -DIMHEX_ENABLE_UNIT_TESTS=OFF \
+        -DIMHEX_PATTERNS_PULL_MASTER=ON \
+        -DIMHEX_ENABLE_LTO=ON \
+        -DIMHEX_USE_GTK_FILE_PICKER=OFF"
+
+# due to dotnet-runtime
+FAIL_ARCH="!(arm64|amd64|armv7hf)"
```
Comment (stdmnpkg) [inline app-editors/imhex/autobuild/defines]: missing \n before eof?

**Comment by stdmnpkg** [comment]
works on my x86 machine
image

**Comment by xtexx** [comment]
LGTM, on amd64

image

File: app-editors/imhex/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,19 @@
+PKGNAME=imhex
+PKGSEC=devel
+PKGDEP__with_dotnet="dotnet-runtime-8.0"
+PKGDES="A hex editor with pattern and yara rule support"
```
Comment (MingcongBai) [inline app-editors/imhex/autobuild/defines]: YARA rules?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-editors/imhex/autobuild/defines (hunk 2 above)
Comment (KexyBiscuit) [reply app-editors/imhex/autobuild/defines]: Maybe capitalize Pattern (language) as well.

File: app-editors/imhex/spec [hunk 3]
```diff
@@ -0,0 +1,5 @@
+VER=1.35.4
+REL=1
```
Comment (KexyBiscuit) [inline app-editors/imhex/spec:2]: New package with REL 1?

**Review by KexyBiscuit — DISMISSED** [review]

File: app-editors/imhex/spec (hunk 3 above)
Comment (MingcongBai) [reply app-editors/imhex/spec:2]: This was due to the application of new revision marking guidelines for topic packages. See commit history.

File: app-editors/imhex/autobuild/defines (hunk 2 above)
Comment (MingcongBai) [reply app-editors/imhex/autobuild/defines]: Seconded:

```
A hex editor with Pattern and YARA Rule support
```

**Review by eatradish — DISMISSED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**